Update Pages “sponsors”

kristinedill opened the pull request:
Automatically generated by Netlify CMS

kylekirkby commented on _pages/sponsors.md:
```diff
@@ -10,34 +10,41 @@ jumbotron:
+          title: Sponsorship Information
           type: image
-          image_content_path: /assets/images/content/lvc21_sponsors.jpg
+          image_content_path: ""
```
@kristinedill - if you're using this `feature_block` section, you'll need and image. Jekyll is failing to build the site since this image path is missing.

kristinedill replied:
Thanks, Kyle. I'll get a new image and try adding it.

kristinedill commented:
Hi @kylekirkby the content on this page that I added in Netlify is not showing up on the preview. Also, I updated the header image with the new LVC21F one and I think it's the wrong size or resolution, but not sure how to tell. 

Also, the same link is still broken https://www.linaro.org/contact/irc/ so it's failing and I'm not sure how to fix that since it's a Linaro page. 
Can you help with this? 

Thanks, 
Kristine

kylekirkby commented:
> Hi @kylekirkby the content on this page that I added in Netlify is not showing up on the preview. Also, I updated the header image with the new LVC21F one and I think it's the wrong size or resolution, but not sure how to tell.
> 
> Also, the same link is still broken https://www.linaro.org/contact/irc/ so it's failing and I'm not sure how to fix that since it's a Linaro page.
> Can you help with this?
> 
> Thanks,
> Kristine

Hi @kristinedill,

I've updated the page in NetlifyCMS. If you're using a slider, the title will need to be in the slider section.

The section was in the wrong format. It needed to be `main_content_row`. That will then display any content you've added in the "main content" section.

I'll take a look at the image. 

Cheers,

Kyle

kylekirkby commented:
@kristinedill - you may need to refresh the page.

https://production-connect-linaro-org-834.ghactions.linaro.org/sponsors/

kristinedill commented:
Hi @EbbaSimpson  or @kylekirkby  can you approve these changes?

Thanks! 

Kristine

kristinedill commented:
@kylekirkby @EbbaSimpson not sure why but I can't merge this even though it's been approved by Ebba.

"Merging is blocked
Merging can be performed automatically once the requested changes are addressed."

kylekirkby commented:
@kristinedill - please try again. I had requested changes but the request was not marked as resolved.

kristinedill commented:
Hi @kylekirkby how do I approve requested changes in the future? 

I tried to publish this just now via Netlify and got an error message: failed to publish. 2 of 2 required status checks are required. at least 1 approving review is required.

Thanks

kylekirkby commented:
> Hi @kylekirkby how do I approve requested changes in the future?
> 
> I tried to publish this just now via Netlify and got an error message: failed to publish. 2 of 2 required status checks are required. at least 1 approving review is required.
> 
> Thanks

@kristinedill please try now. It looks like if I have requested changes, I have to approve that the requested changes have been made. It's rare that changes are requested but if I do request them in the future, I'll make sure to approve once it's been resolved.
